manifests/init.pp

```diff
@@ -391,14 +391,10 @@
 
   case $facts['os']['name'] {
```

Comment:
technically, the whole case could be removed and the `gnupg` ensure could be made unconditional?

Reply:
Yep, same with the change in `key.pp`.